src/shared/constants.ts

```diff
@@ -79,7 +79,8 @@ export interface NetworkConfiguration {
   };
 }
 
-export const HIRO_API_KEY = 'leather_i2LDUpJTyVZcLz51EoY57QjYr8fx8vvX';
+// Disabling temporarily, but we might need again in the future
```

Comment:
I was thinking maybe we should remove the key altogether? 

It will be in the GIT history anyway though so not a big deal I guess

Reply:
Yeah, I can remove, np. Fine to ref code later.